docs/visualization/visualization_api/callback_information.md

```diff
@@ -1,20 +1,23 @@
 # Callback Information
 
+To investigate with fine granularity, the behavior of callbacks should be visualized in detail.
 CARET can visualize the callback frequency, period, and latency.
-Execute these commands in advance.
+This document describes sample visualization scripts for them.
+Execute these commands before each section.
 
 ```python
 from caret_analyze.plot import Plot
 from caret_analyze import Application, Architecture, Lttng
 from bokeh.plotting import output_notebook, figure, show
 output_notebook()
-#output_notebook() is needed only when you run the script on Jupyter Notebook
 arch = Architecture('yaml', '/path/to/architecture_file')
 lttng = Lttng('/path/to/trace_data')
 app = Application(arch, lttng)
 ```
 
-## Execution frequency
+Note: The `output_notebook()` is needed only when running the script on Jupyter Notebook.
```

Comment:
@takeshi-iwanari 
I adapted your comment.
https://github.com/tier4/CARET_doc/pull/95#discussion_r999183416
> output_notebook() is needed only for jupyter notebook.
> It may be better to add note like either of the followings (maybe in the top page of visualization page):
> output_notebook() is needed only when you run the script on Jupyter Notebook
> There are sample scripts for Jupyter Notebook

I add only add `The `output_notebook()` is needed only when running the script on Jupyter Notebook.` because the code depending on jupyter-notebook is only `output_notebook()`.
I think it is OK to add Note only output_notebook().